docs/build_docs/source/errors/index.rst

```diff
@@ -51,9 +51,50 @@ in your input file have correct values. Remember that ligands needs to have a un
 Connections Error
 ------------------
 
-This error indicates the the PDB file is missing the connectivity section. To resolve the issue, you should import the PDB in Schrödinger Maestro and preprocess it, launching the Protein Preparation Wizard. 
+This error indicates the the PDB file is missing the connectivity section. To resolve the issue, you should import the PDB in Schrödinger Maestro and preprocess it, launching the Protein Preparation Wizard.
```

Comment:
Is it an error or a warning? We should mention that when using peleffy the connectivity is required and in case it is missing, we use Schrodinger's prepwizard script.